### PR TITLE
add navigation

### PR DESCRIPTION
**Pull request by Sukhpreet92**

### CODE REVIEW

File: components/Card.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import styled from "styled-components";
 import Image from "next/image";
-import deleteIcon from "../public/images/deleteIcon.svg";
+import deleteIcon from "../public/images/deleteIcon.png";
```
Comment (philmetscher) [inline components/Card.js:3]: SVG is usually better than PNG. Why did you change that?

File: components/Card.js [hunk 2]
```diff
@@ -32,9 +32,12 @@ const CardContainer = styled.li`
 `;
 
 const DeleteButton = styled.button`
-  background-color: pink;
-  width: 2em;
-  height: 2.2em;
+  background-color: white;
```
Comment (philmetscher) [inline components/Card.js]: Browsers interpret values like `white` differently. Please always use Hex, RGB or HSL values

Comment (philmetscher) [reply components/Card.js]: Look at your code and correct everything in CSS related to the color values :)

File: components/Navigation.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      <UlStyledList>
+        <StyledList>
+          <SyledHomeButton onClick={() => router.push("/")} type="button">
+            <Image src={homeIcon} alt="Go to Homepage" />
+          </SyledHomeButton>
+        </StyledList>
+        <StyledList>
+          <StyledAddButton onClick={() => router.push("Create")} type="button">
+            <Image src={addIcon} alt="Go to Createpage" />
+          </StyledAddButton>
```
Comment (philmetscher) [inline components/Navigation.js]: Why do you use buttons for this? Tip: `next/link`

**Review by philmetscher — CHANGES_REQUESTED** [review]

File: components/Card.js (hunk 2 above)
Comment (Sukhpreet92) [reply components/Card.js]: > Look at your code and correct everything in CSS related to the color values :)

I will change it to hex codes 👍

File: components/Card.js (hunk 1 above)
Comment (Sukhpreet92) [reply components/Card.js:3]: > SVG is usually better than PNG. Why did you change that?

svg loading was damn slow in inspect modus.

File: components/Navigation.js (hunk 3 above)
Comment (Sukhpreet92) [reply components/Navigation.js]: > Why do you use buttons for this? Tip: `next/link`

I will look into `next/link`  :eyes:

File: pages/Create.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+`;
+const HabitForTodayList = styled.ul`
+  font-size: 1.5em;
+  color: pink;
+`;
+
+const HabitDone = styled.ul`
+  font-size: 1.2em;
+  filter: blur(0.04em);
+`;
```
Comment (AHen-C0de) [inline pages/Create.js]: I think this are still copy left overs? They dont appear in the file ;)

**Review by AHen-C0de — CHANGES_REQUESTED** [review]
Only a minor comment from my site!

File: components/Navigation.js [hunk 5]
```diff
@@ -2,30 +2,29 @@ import styled from "styled-components";
...
 import addIcon from "../public/images/addIcon.png";
-import { useRouter } from "next/router";
+import Link from "next/link";
 
 export default function Navigation() {
-  const router = useRouter();
   return (
     <StyledNavigation>
       <UlStyledList>
         <StyledList>
```
Comment (philmetscher) [inline components/Navigation.js:11]: I think you should take another look at what you call your styled components
Suggestion:
**You don't need `Ul` here**
`UlStyledList`: `List` or `StyledList`
**That's an Entry, not a List**
`StyledList`: `ListEntry` or `StyledListEntry`

File: components/Navigation.js [hunk 6]
```diff
@@ -2,30 +2,29 @@ import styled from "styled-components";
...
-          <SyledHomeButton onClick={() => router.push("/")} type="button">
+          <Link href="/">
             <Image src={homeIcon} alt="Go to Homepage" />
-          </SyledHomeButton>
+          </Link>
```
Comment (philmetscher) [inline components/Navigation.js:14]: u don't need styling for the Links?
Otherwise you could write
```javascript
<Link href="/" passHref>
  <NavLink aria-label="Go to Homepage">
    <Image src={homeIcon} alt="Go to Homepage" />
  </NavLink>
</Link>

const NavLink = styled.a`
  width: 3em;
  height: 3em;
  border: 0em;
  background-color: transparent;

  &:hover {
    cursor: pointer;
    filter: invert(1);
  }
`;
```

**Review by philmetscher — APPROVED** [review]
